packages/flutter/lib/src/painting/binding.dart

```diff
@@ -84,13 +84,15 @@ mixin PaintingBinding on BindingBase, ServicesBinding {
   Future<ui.Codec> instantiateImageCodec(Uint8List bytes, {
     int cacheWidth,
     int cacheHeight,
+    bool allowUpscaling = false,
```

Comment:
Can you add a sentence about this parameter to the doc comment above?

Reply:
I updated the docstring in general here - there were some old no-longer-existant things in there and some formatting issues, as well as something that's changed about aspect ratio preservation.  PTAL